persist/fs/msgpack/decoder.go

```diff
@@ -269,8 +328,8 @@ func (dec *Decoder) decodeLogEntry() schema.LogEntry {
 		return emptyLogEntry
 	}
 	var logEntry schema.LogEntry
-	logEntry.Create = dec.decodeVarint()
 	logEntry.Index = dec.decodeVarUint()
+	logEntry.Create = dec.decodeVarint()
```

Comment:
could you put a blurb explaining the rationale of ordering somewhere. Don't want the nuance to get lost the next time someone comes and tries to make changes to the commit log format

Reply:
I'll add a comment to the encoder code